usl_pipeline/usl_pipeline/formats/geotiff_loader.py

```diff
@@ -0,0 +1,39 @@
+import rasterio.features
+
+
+def load_elevation_from_geotiff(
+    file_path: str, load_data: bool = False, result_nodata: float = None
+):
+    """
+    Loading elevation raster data from GeoTIFF file.
+    :param file_path: path to a file to load from
```

Comment:
Let's capitalize and add punctuation to these ala
https://numpydoc.readthedocs.io/en/latest/format.html#parameters

Reply:
I decided to follow https://google.github.io/styleguide/pyguide.html#383-functions-and-methods. Let me know if you prefer different style.

Reply:
Works for me as long as it's compatible with black and flake8 and all. 
Can you add a line or two to
https://www.notion.so/Software-Development-Life-Cycle-05b5181505cd4cd298299b6cd6c745ad
about this style guide?

Reply:
Done